storage.bs

```diff
@@ -254,12 +283,22 @@ given a <a for=map>value</a> when a <a>storage shelf</a> is
 <var>environment</var>, return the result of running <a>obtain a storage shelf</a> with the user
 agent's <a for="user agent">storage shed</a>, <var>environment</var>, and "<code>local</code>".
```

Comment:
It's a little unclear why "local" gets special treatment.

Reply:
This is not an exported algorithm. I added this for a number of internal callers.

Reply:
In that case it seems like it'd be even nicer to just let the internal callers pass "local" as an argument instead of as part of the algorithm name.

Reply:
They would also have to pass in the storage shed. I think it's worth having the abstraction (and this was already approved as part of a prior PR).

Reply:
Well, just adding my two cents as a reader that it makes things pretty confusing. Up to you whether to fix it or not.

Reply:
Why is this case confusing but other similar cases are not?

Reply:
All other cases appear to be symmetrical, with shortcut definitions for both local and session. If there are ones I missed where local is special (but not called out with a note as to why it's special), then I agree those are also confusing.